src/data/fintype/basic.lean

```diff
@@ -1118,10 +1118,7 @@ instance finset.fintype [fintype α] : fintype (finset α) :=
 fintype.of_equiv _ (equiv.subtype_injective_equiv_embedding α β)
 
 instance [decidable_eq α] [fintype α] {n : ℕ} : fintype (sym.sym' α n) :=
-begin
-  unfold sym.sym' vector.perm.is_setoid,
-  exact quotient.fintype _,
-end
+by { unfold sym.sym' vector.perm.is_setoid, exact quotient.fintype _ }
```

Comment:
I worry this will result in a fintype with weird`eq.rec`s

bors d-

I'll try this out in gitpod shortly

Reply:
I think the best thing to do would be to add the missing decidable instances

Reply:
Making `vector.perm.is_setoid` `[reducible]` (and adding a comment about `note [reducible non-instances]`) solves the problem.

Reply:
```suggestion
quotient.fintype _
```